src/Block.cpp

```diff
@@ -238,6 +238,7 @@ Block::~Block(void)
 	deleted_stms.clear();
 
 	local_vars.clear();
+	local_var_set.clear();
```

Comment:
I don't think we need to maintain two collections of local variables. Local variables as a set is preferred.